Blazor prerendering language update (#15427)
aspnet/AspNetCore.Docs: aspnetcore/blazor/handle-errors.md

Blazor components can be prerendered using `Html.RenderComponentAsync` so that their rendered HTML markup is returned as part of the user's initial HTTP request. This works by:

> * Creating a new circuit to update the prerendered components that are part of the same page.

Reviewer: Hmm, we don't really update the components. We render them into the page as part of the response. Honestly, the relationship between the circuit and the prerendered components isn't super relevant.

Reviewer: How about this?

```suggestion
* Creating a new circuit for all of the prerendered components that are part of the same page.
```

* Generating the initial HTML.

> * Treating the circuit as `disconnected` until the user's browser establishes a SignalR connection back to the same server. When the connection is established, interactivity on the circuit is resumed and the components' HTML markup is updated.

Reviewer: This looks good.
